sdk/core/azure-core-tracing-opentelemetry/pom.xml

```diff
@@ -50,6 +50,18 @@
       <version>0.2.0</version> <!-- {x-version-update;io.opentelemetry:opentelemetry-sdk;external_dependency} -->
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-core-test</artifactId>
+      <version>1.1.0</version> <!-- {x-version-update;com.azure:azure-core-test;dependency} -->
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-core-http-netty</artifactId>
```

Comment:
Is anything actually using this dependency?

Reply:
Used for creating a default client [here](https://github.com/Azure/azure-sdk-for-java/pull/7488/files#diff-9c6faa2122f174f015c3e0c7fe8e623bR86).